CMakeLists.txt

```diff
@@ -19,18 +19,21 @@ endif()
 
 set(RADIX_INCLUDE_DIRS "")
 
-find_package(CXX14 REQUIRED)
-set(CMAKE_CXX_FLAGS "${CXX14_FLAGS} ${CMAKE_CXX_FLAGS}")
+set (CMAKE_CXX_STANDARD_REQUIRED ON)
+set (CMAKE_CXX_STANDARD 17)
+set (CMAKE_CXX_FLAGS " -DNOMINMAX --std=c++1z ${CMAKE_CXX_FLAGS}")
+#find_package(CXX14 REQUIRED)
+#set(CMAKE_CXX_FLAGS "${CXX14_FLAGS} ${CMAKE_CXX_FLAGS}")
 
 find_package(OpenGL REQUIRED)
 list(APPEND RADIX_INCLUDE_DIRS ${OPENGL_INCLUDE_DIR})
 
 set(RADIX_LIBRARIES ${OPENGL_gl_LIBRARY})
 
 find_package(PkgConfig REQUIRED)
-pkg_search_module(EPOXY REQUIRED epoxy)
-list(APPEND RADIX_INCLUDE_DIRS ${EPOXY_INCLUDE_DIRS})
-list(APPEND RADIX_LIBRARIES ${EPOXY_LIBRARIES} )
+#pkg_search_module(EPOXY REQUIRED epoxy)
```

Comment:
@GlPortal/push-developers @acterhd  I don't think we want to switch to glad like this but rather use epoxy whenever it is available.